Tests/scs-compatible-kaas.yaml

```diff
@@ -6,9 +6,12 @@ variables:
 modules:
   - id: cncf-k8s-conformance
     name: CNCF Kubernetes conformance
-    url: https://github.com/cncf/k8s-conformance/tree/master
+    url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0210-v2-k8s-version-policy.md # TODO:!!! this standard mentions cncf but is is the right reference?
+    run:
+      - executable: ./kaas/plugin/run_sonobuoy_executor.py
+        args: -k {kubeconfig} -r {result_dir_name} -c 'cncf-conformance'
     testcases:
-      - id: cncf-k8s-conformance
+      - id: cncf-conformance
```

Comment:
I don't think this should be part of this PR either. Apart from that:

- the original URL was correct; there is (by design) no SCS-specific url for CNCF k8s conformance
- I don't see the need for changing the testcase id

Reply:
Removed it from this PR.